src/test/resources/org/jabref/logic/crawler/study.yml

```diff
@@ -0,0 +1,16 @@
+authors:
+  - Jab Ref
+studyName: TestStudyName
```

Comment:
I think, yml convention is to use hyphen to connect words, e.g. `study-name`. (maybe `name` is already sufficient?)